### ISSUE TITLE
Add standard CSS variables

### ISSUE THREAD

**Issue by krambuhl** [connectivedx/fuzzy-chainsaw#103]
We should provide some standard set of custom variables that can be useful as a starting point.  Things like colors, custom media breakpoints and such.

**Comment by devinrayolsen** [comment]
//variables/typography.css
```
:root {
	--baseFontSize: 16;
	--baseFontPercentage: calc(calc(var(--baseFontSize) / 16) / 100%);
	--lineHeight: 1.5;
	--baseFontFamily: 'Source Sans Pro', Helvetica, Arial, sans-serif;
	--listBulletSize: 24;
	--baseFont: var(--baseFontPercentage) / var(--lineHeight);
}
```

//tags/PageRoot/PageRoot.css
```
html {
	font: var(--baseFont) var(--baseFontFamily);
}
```

**Comment by stoff** [comment]
can you get this up on a branch?

**Comment by krambuhl** [comment]
absolutely!  make a pull request.

**Comment by drolsen** [comment]
Pull request made. Once we get this in I can update grids branch to be responsive.

**Comment by krambuhl** [comment]
I've merged #112 that adds standard variables. Closing this for now, reopen if we still have useful variables missing.
